compiler/data/class-data.h

```diff
@@ -58,6 +58,7 @@ class ClassData : public Lockable {
   bool really_used{false};
   bool is_tl_class{false};
   bool has_custom_constructor{false};
+  bool has_inherited_constructor{false}; // set to true if derived class gets its constructor from the base class
```

Comment:
To be honest, I suppose, that this field can be avoided.
FunctionData has the `is_auto_inherited` field. Maybe, play around that field for constructors? Instead of adding a new one to ClassData

Reply:
Good idea. Fixed.